docs/source/developer-docs/contrib_guide_code.rst

```diff
@@ -109,3 +122,37 @@ be able to lend you assistance.
 Contribution Workflow
 ---------------------
 .. image:: ../_static/_images/contribution_workflow.png
+
+
+
+Core Developer Workflow
+-----------------------
+
+It is interesting to know how the core developer works in order to understand
```

Comment:
To help you understand our review process by core developers after you submit a pull request, here's a guide that outlines the general process (specifics may vary a bit across our repositories). Here is an example for Jupyter notebook 4.x: